DEVELOPING.md

```diff
@@ -0,0 +1,129 @@
+The default  test systems are configured in [liquibase.sdk.yaml](liquibase-extension-testing/src/main/resources/liquibase.sdk.yaml). 
+When that line in your test is hit, the TestSystemFactory will use a running system if available, or start a new system if needed. 
+Any newly-started systems will be available for the lifetime of **all** the tests to be shared between them and be shut down at the end of the entire run.
+
+The base TestSystem class is designed to work for more than just Databases, so if you know you are connecting to a database you will want to cast it to a DatabaseTestSystem in order to call methods like `getConnection()` for use in connecting to it. 
+For example:
+```java
+@Shared
+private DatabaseTestSystem mysql = 
+    (DatabaseTestSystem) Scope.getCurrentScope().getSingleton(TestSystemFactory.class).getTestSystem("mysql")
+```
+
+When TestSystems need to be started, Liquibase generally relies on Docker instances. For time when docker images are not available or are unneeded, they can also connect to an external database via a `url` setting. 
+
+When using docker instances, if we want to add support for a new database we want to test against, we need to create a new test class for which we will need to extend from `DatabaseTestSystem` and override
+`createContainerWrapper()` method, as displayed below:
+```java
+public class FirebirdTestSystem  extends DatabaseTestSystem {
+
+    public FirebirdTestSystem(Definition definition) {
+        super(definition);
+    }
+
+    @Override
+    protected DatabaseWrapper createContainerWrapper() throws Exception {      
+        return new DockerDatabaseWrapper(new FirebirdContainer(
+                DockerImageName.parse(getImageName()).withTag(getVersion()))
+                .withDatabaseName(getCatalog())
+                .withUsername(getUsername())
+                .withPassword(getPassword()),
+                this
+        );
+    }
+}
+```
+
+Additionally, we need to add the test dependency for the DB we are adding support for in the `pom.xml` file from **liquibase-extension-testing** module:
+```xml
+<dependency>
+    <groupId>org.firebirdsql</groupId>
+    <artifactId>firebird-testcontainers-java</artifactId>
+    <version>1.2.0</version>
+</dependency>
+```
+
+#### Controlling which systems are tested against
+
+Even though systems are able to be automatically brought up, tests against databases are slow. Therefore, exactly which test systems should be tested against is controllable by you.
+By default, Liquibase will only test against in-memory databases (h2, hsqldb, sqlite) because they are fast. Any tests which ask for other systems will automatically be marked as "ignored".
+
+To change which tests run and which are ignored, set the `liquibase.sdk.testSystem.test` attribute to be a comma-separated list. This can be most easily set in your liquibase.sdk.local.yml file (see below).
```

Comment:
Edit to line 77 to simply state the recommended method to change which tests run.

"The `liquibase.sdk.testSystem.test` attribute is a comma-separated list specified in the liquibase.sdk.local.yml (see below). "

Reply:
Done.